cfme/networks/provider/nuage.py

```diff
@@ -44,6 +46,27 @@ class events(Tab, BeforeFillMixin):
         validate = Button('Validate')
 
 
+class ValidateStatsMixin(object):
+    def validate_stats(self, host_stats, ui=False):
+        """ Validates that the detail page matches the subnet's inventory.
+
+        Args:
+            host_stats: dictionary of values to be compered to UI values,
+            where keys have the same names as NuageSubnet ui functions
+
+        This method gets host stats and matches them against the UI. An exception will
+        be raised if the stats retrieved from the UI do not match those from host stats.
```

Comment:
_An exception will be raised_ - please be specific. If I wanted to catch the exception I'd need to know excpetion class.